install-roles/README.md

```diff
@@ -3,23 +3,60 @@
 This directory contains the minimal role and clusterRole that must be assigned to
 a user in order to install fusion 5 with helm v3.
 
+First, use the `customize_fusion_values.sh` script for creating your fusion values yaml file.
+
 To use these role in a cluster, as an admin user first create the namespace that you wish to
-install fusion into:
+install fusion and label it into:
+
+```
+kubectl create namespace fusion-namespace
+kubectl label namespace fusion-namespace seldon.io/controller-id=fusion-namespace
+```
+
+Install the required CRDs as an admin user, for this we are going to download the fusion chart:
+
 ```
-k create namespace fusion-namespace
+helm fetch lucidworks/fusion --version <fusion-version> --untar
+find . -iname crds -type d -exec kubectl apply -f {} \;
 ```
+
 Apply the `role.yaml` and `cluster-role.yaml` files to that namespace
 
 ```
-k apply -f cluster-role.yaml
-k apply -f --namespace fusion-namespace role.yaml
+kubectl apply -f cluster-role.yaml
```

Comment:
Do we use `k` in the rest of the documentation in this repo? Should we keep it as k for consistency?

Reply:
Updated readme replacing k with kubectl